source/test/cli.ts

```diff
@@ -94,17 +115,26 @@ test('cli typings and files flags', async t => {
 	const {exitCode, stderr} = t.throws<ExecaError>(() => execa.commandSync(`dist/cli.js -t ${typingsFile} -f ${testFile}`));
 
 	t.is(exitCode, 1);
-	t.true(stderr.includes('✖  5:19  Argument of type number is not assignable to parameter of type string.'));
+	verifyCli(t, stderr, [
+		'✖  5:19  Argument of type number is not assignable to parameter of type string.',
+		'',
+		'1 error',
+	]);
 });
 
 test('tsd logs stacktrace on failure', async t => {
-	const {exitCode, stderr, stack} = await t.throwsAsync<ExecaError>(execa('../../../cli.js', {
+	const {exitCode, stderr} = await t.throwsAsync<ExecaError>(execa('../../../cli.js', {
 		cwd: path.join(__dirname, 'fixtures/empty-package-json')
 	}));
 
 	t.is(exitCode, 1);
-	t.true(stderr.includes('Error running tsd: JSONError: Unexpected end of JSON input while parsing empty string'));
-	t.truthy(stack);
+
+	verifyCli(t, stderr, [
+		'Error running tsd:',
+		'JSONError: Unexpected end of JSON input while parsing empty string',
+		// TODO: check that stack matches without checking for exact filepath
+		// would have to match in CI and locally
+	], {startLine: 0});
```

Comment:
@skarab42 how do you think we should test the error stack? I moved the CLI verification you added from `test/diff.ts` into `utils`, but it only tests the lines you've provided.